clustertask/stakater-sonarqube-scanner/templates/clustertask.yaml

```diff
@@ -0,0 +1,227 @@
+apiVersion: tekton.dev/v1beta1
+kind: ClusterTask
+metadata:
+  name: {{ include "stakater-sonarqube-scanner.name" . }}
+spec:
+  description: >-
+    The following task can be used to perform static analysis on the source code
+    provided the SonarQube server is hosted
+    
+    SonarQube is the leading tool for continuously inspecting the Code Quality and Security
+    of your codebases, all while empowering development teams. Analyze over 25 popular
+    programming languages including C#, VB.Net, JavaScript, TypeScript and C++. It detects
+    bugs, vulnerabilities and code smells across project branches and pull requests.
+
+  workspaces:
+    - name: source
+  params:
+    - name: sonar_host_url
+      description: Host URL where the sonarqube server is running
+      default: ""
+    - name: sonar_project_key
+      description: Project's unique key
+      default: ""
+    - name: sonar_login
+      description: Sonar username
+      default: ""
+    - name: sonar_password
+      description: Sonar password
+      default: ""
+  steps:
+    - name: sonar-properties-create
```

Comment:
@AsfaMumtaz name with -? doesn't seem to be following our conventions

Reply:
I think the - convention is fine for step name. I've changed all params to follow the underscore + capitalized convention